examples/aei_from_events/aei_from_events.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+
+all_cat = ['festivals', 'school-holidays', 'airport-delays', 'sports',
+           'performing-arts', 'concerts', 'public-holidays', 'community',
+           'observances', 'severe-weather', 'expos', 'conferences',
+           'health-warnings', 'academic', 'disasters', 'politics', 'terror']
+
+nonattended = ['public-holidays', 'observances', 'school-holidays', 'academic']
+
+attended = ['festivals', 'sports', 'performing-arts',
+            'concerts', 'community', 'expos', 'conferences']
+
+unscheduled = ['airport-delays', 'health-warnings', 'disasters', 'politics', 'terror', 'severe-weather']
+
+from datetime import datetime, timedelta
+
+def gen_cal(st, ed):
+    clean_start = st.replace('T', ' ').replace('Z', '')
+    start_dt = datetime.strptime(clean_start, '%Y-%m-%d %H:%M:%S')
```

Comment:
I am not sure that you need your clean_start step, you might be able to make your strptime format to deal with that.

Reply:
Referencing [this](https://stackoverflow.com/questions/29284850/datetime-strptime-set-format-to-ignore-trailing-part-of-string) post to defend my choice!